packages/date-picker/src/FilterDateRangePicker/components/DateRangeInputField/DateRangeInputField.tsx

```diff
@@ -71,10 +71,23 @@ export const DateRangeInputField = React.forwardRef<
     const inputRangeEndRef = customRefObject?.inputRangeEndRef
 
     const descriptionId = `${id}--field-message`
-    const errorMessageId = `${id}--error-message`
 
-    const inputDescribedBy = validationMessage
-      ? `${errorMessageId} ${descriptionId}`
+    // Date Start aria labels
+    const dateStartErrorMessageId = validationMessage?.dateStart
+      ? `${id}--date-start-error-message`
+      : undefined
+
+    const dateStartInputDescribedBy = dateStartErrorMessageId
+      ? `${dateStartErrorMessageId} ${descriptionId}`
```

Comment:
We still need to include the error message within the aria-describeby as aria-errormessage is not yet compatible for "pertinent" messages  see [here](https://github.com/cultureamp/kaizen-discourse/issues/29#issuecomment-1449290697)

I still wanted to add them as the basic functionality is supported